### PR TITLE
chore(lazy-loading): initialize lazy-loading example

### PR DESCRIPTION
**Pull request by skipjack**
Also had this one sitting around. Pretty sure it does line up with the finalized _Lazy Loading_ guide. If you see any discrepancies, let me know.

### CODE REVIEW

File: examples/lazy-loading/src/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    element.appendChild(br);
+    element.appendChild(button);
+ 
+    // Note that because a network request is involved, some indication
+    // of loading would need to be shown in a production-level site/app.
+    button.onclick = e => import(/* webpackChunkName: "print" */ './print').then(module => {
+      var print = module.default;
+
+      print();
+    })
```
Comment (TheDutchCoder) [inline examples/lazy-loading/src/index.js]: Missing ;

File: examples/lazy-loading/src/print.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+console.log('The print.js module has loaded! See the network tab in dev tools...');
+
+export default () => {
+  console.log('Button Clicked: Here\'s "some text"!');
+}
```
Comment (TheDutchCoder) [inline examples/lazy-loading/src/print.js]: Missing ;

**Review by TheDutchCoder — CHANGES_REQUESTED** [review]
Add the missing semicolons and I'll merge this in. We can alter the examples in another PR later on, if they're not aligned with the Guides yet/anymore.

**Comment by skipjack** [comment]
@TheDutchCoder updated. Side note: this is an example where "Squash and Merge" would be perfect to get rid of all those "... semicolon" commits. They would just create noise on master and aren't very helpful to have in the history.

**Comment by TheDutchCoder** [comment]
Gotcha!

**Pull request merged**